### PR TITLE
Expand microGalaxy September meeting description

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: _events/2021-09-14-microgalaxy-call.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ starts: 2021-09-23
 ends: 2021-09-23
 location: online
 supporters:
-external:
+external: https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit?usp=sharing
 ---
 
-The microGalaxy community focus on the biology of microbes. The call will start a 2 pm CEST and will last for 2 hours, the topic will be "Citizen-led science and education".
+The microGalaxy community is a [new community focusing on microbial analysis within Galaxy community](https://docs.google.com/document/d/1btaPVcrE9Hr6gLQUsqdxHKX9IvtWaWINkLZ3_NcOhe8/edit?usp=sharing). The second meeting of microGalaxy will happen on [September 23rd at 2 pm CEST](https://arewemeetingyet.com/Berlin/2021-09-23/14:00/microGalaxy) and will last for 2 hours. The topic will be "Citizen-led science and education", with 2 talks and discussions around and beyond this topic. Anyone interested is welcomed to join: details will be added in the [meeting minutes](https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit?usp=sharing)
```
Comment (beatrizserrano) [inline _events/2021-09-14-microgalaxy-call.md:12]:
```suggestion
The microGalaxy community is a [new community focusing on microbial analysis within Galaxy community](https://docs.google.com/document/d/1btaPVcrE9Hr6gLQUsqdxHKX9IvtWaWINkLZ3_NcOhe8/edit?usp=sharing). The second meeting of microGalaxy will happen on [September 23rd at 2 pm CEST](https://arewemeetingyet.com/Berlin/2021-09-23/14:00/microGalaxy) and will last for 2 hours. The topic will be "Citizen-led science and education", with 2 talks and discussions around and beyond this topic. Anyone interested is welcome to join: details will be added in the [meeting minutes](https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit?usp=sharing)
```

Comment (beatrizserrano) [inline _events/2021-09-14-microgalaxy-call.md:12]:
```suggestion
microGalaxy is a [new community focusing on microbial analysis with Galaxy](https://docs.google.com/document/d/1btaPVcrE9Hr6gLQUsqdxHKX9IvtWaWINkLZ3_NcOhe8/edit?usp=sharing). The second meeting of microGalaxy will happen on [September 23rd at 2 pm CEST](https://arewemeetingyet.com/Berlin/2021-09-23/14:00/microGalaxy) and will last for 2 hours. The topic will be "Citizen-led science and education", with 2 talks and discussions around and beyond this topic. Anyone interested is welcomed to join: details will be added in the [meeting minutes](https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit?usp=sharing)
```

File: _events/2021-09-14-microgalaxy-call.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ starts: 2021-09-23
 ends: 2021-09-23
 location: online
 supporters:
-external:
+external: https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit?usp=sharing
 ---
 
-The microGalaxy community focus on the biology of microbes. The call will start a 2 pm CEST and will last for 2 hours, the topic will be "Citizen-led science and education".
+The microGalaxy community is a [new community focusing on microbial analysis within Galaxy community](https://docs.google.com/document/d/1btaPVcrE9Hr6gLQUsqdxHKX9IvtWaWINkLZ3_NcOhe8/edit?usp=sharing). The second meeting of microGalaxy will happen on [September 23rd at 2 pm CEST](https://arewemeetingyet.com/Berlin/2021-09-23/14:00/microGalaxy) and will last for 2 hours. The topic will be "Citizen-led science and education", with 2 talks and discussions around and beyond this topic. Anyone interested is welcome to join: details will be added in the [meeting minutes](https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit?usp=sharing)
```
Comment (bebatut) [inline _events/2021-09-14-microgalaxy-call.md:12]:
```suggestion
microGalaxy is a [new community focusing on microbial analysis with Galaxy](https://docs.google.com/document/d/1btaPVcrE9Hr6gLQUsqdxHKX9IvtWaWINkLZ3_NcOhe8/edit?usp=sharing). The second meeting of microGalaxy will happen on [September 23rd at 2 pm CEST](https://arewemeetingyet.com/Berlin/2021-09-23/14:00/microGalaxy) and will last for 2 hours. The topic will be "Citizen-led science and education", with 2 talks and discussions around and beyond this topic. Anyone interested is welcome to join: details will be added in the [meeting minutes](https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit?usp=sharing)
```

**Comment by beatrizserrano** [comment]
thanks for the update @bebatut

**Pull request merged**